client/a8c-for-agencies/components/items-dashboard/item-preview-pane/item-preview-pane-header/index.tsx

```diff
@@ -78,6 +78,14 @@ export default function ItemPreviewPaneHeader( {
 							) : (
 								itemData.subtitle
 							) }
+
+							{ extraProps && extraProps.subtitleExtra ? (
+								<span>
+									<extraProps.subtitleExtra />
+								</span>
+							) : (
+								''
+							) }
```

Comment:
Nitpick (non-blocking): This can be simplified
```suggestion
							{ extraProps && extraProps.subtitleExtra && ( <span><extraProps.subtitleExtra /></span> ) }
```